Command/DebugRouterCommand.php

```diff
@@ -49,7 +50,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $router = $this->routers[$rname];
 
         $collection = $router->getCollection();
-        $table = $this->getHelperSet()->get('table');
+        $table = new Table($output);
```

Comment:
We should keep both since 2.x is not deprecated in favor of 3.0, could you check the symfony version and use the appropriate way ?